js/ElementRendererProvider.js

```diff
@@ -366,19 +366,21 @@ export default class ElementRendererProvider {
       const defaultPadding = 0;
       const padding = config.padding || defaultPadding;
       let nextLeft = 0;
-      const arrowRight = document.createElement('div');
-      const arrowLeft = document.createElement('div');
+      const arrowRight = document.createElement('button');
```

Comment:
Please verify is across all relevant browsers (including IE11)

Reply:
Will do, thanks! BTW we are not maintaining IE11 anymore

Reply:
@meirotstein verified 
on Mac - Safari, Chrome, Firefox
on Windows - IE11, Edge(latest), Chrome(latest)

Reply:
Thanks! it is true that we are not maintaining IE11, but I don't want us to break existing functionality